Dockerfile

```diff
@@ -10,7 +10,12 @@
 
 FROM alpine:3.10.2
```

Comment:
If we're updating all the deps, may as well update the base image to the latest (I think it's 3.11.3 these days).